src/Feedback.Analyzer.Backend/Feedback.Analyzer.Application/Common/Workflows/Events/ExecuteWorkflowSinglePromptEvent.cs

```diff
@@ -0,0 +1,10 @@
+using Feedback.Analyzer.Domain.Common.Events;
+
+namespace Feedback.Analyzer.Application.Common.Workflows.Events;
+
+public class ExecuteWorkflowSinglePromptEvent : Event
+{
+    public Guid WorkflowId { get; set; }
+    
+    public Guid PromptId { get; set; }
+}
```

Comment:
Add docs and change to record, change `Event` to record as well, change setters to initializers ( set -> init )